dist/Leaflet.fullscreen.js

```diff
@@ -1,6 +1,5 @@
 L.Control.Fullscreen = L.Control.extend({
     options: {
-        position: 'topleft',
         title: {
             'false': 'View Fullscreen',
```

Comment:
Hmm, is it possible to change these values by passing an option to `L.map()` somehow?